### PR TITLE
Create mexico-​chiapas-tuxtla.json

### PR DESCRIPTION
**Pull request by MounaBelaid**

### CODE REVIEW

File: data/chapters/mexico-​chiapas-tuxtla.json [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+    "urlname": "rladies-tuxtla-gutierrez",
+    "status": "prospective",
+    "country": "mexico",
+    "city": "tuxtla-gutierrez",
+    "social_media": {
+      "meetup": "rladies-tuxtla-gutierre",
+      "email": "tuxtla@rladies.org"
+    },
+    "organizers": {
+      "current": ["comma separated", "Amin Siman Israel"]
```
Comment (drmowinckels) [inline data/chapters/mexico-​chiapas-tuxtla.json:12]:
```suggestion
      "current": ["Amin Siman Israel"]
```

Comment (drmowinckels) [reply data/chapters/mexico-​chiapas-tuxtla.json:12]: @MounaBelaid awesome if you can remove the "comma separated" part i the future :)

**Review by drmowinckels — CHANGES_REQUESTED** [review]

File: data/chapters/mexico-​chiapas-tuxtla.json (hunk 1 above)
Comment (MounaBelaid) [reply data/chapters/mexico-​chiapas-tuxtla.json:12]: Yes, sure! Thanks.

**Review by drmowinckels — APPROVED** [review]

**Comment by rivaquiroga** [comment]
@MounaBelaid, @drmowinckels, maybe the pull request for creating the json file for new chapters should be kept as a draft until we complete the process. The meetup account and the email for this chapter has not been created because we are not yet in [that step](https://github.com/rladies/new-chapters-onboarding/issues/6). I'll update the templete so the process is better explained!

**Pull request merged**